### PR TITLE
KG - Create New Participant Modal Bug Fix

### PR DESCRIPTION
**Pull request by kyle-glick**

### CODE REVIEW

File: app/assets/javascripts/participants.js.coffee [hunk 1]
```diff
@@ -11,6 +11,15 @@ $ ->
       type: 'GET'
       url: "/participants/#{participant_id}/change_arm?arm_id=#{arm_id}"
 
+  $(document).on 'click', '.new-participant', ->
+    data = 
+      'protocol_id' : $(this).attr('protocol_id')
+
+    $.ajax
+      type: 'GET'
+      url: "/participants/new"
```
Comment (goleksiak) [inline app/assets/javascripts/participants.js.coffee]: I think we should set the :format on urls. So: url: "/participants/new.js"

File: app/views/protocols/_participant_list.html.haml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 %h3= t(:participant)[:list]
 #custom-toolbar
-  = link_to t(:participant)[:new_participant], new_participant_path(protocol_id: protocol.id), {class: 'btn btn-default btn-primary', id: 'new_participant_button', type: 'button', data: {toggle: 'modal', target: '#modal_place'}, remote: true}
+  %button.btn.btn-primary.new-participant{type: "button", protocol_id: protocol.id, aria: {label: "Create Participant"}}=t(:participant)[:new_participant]
```
Comment (goleksiak) [inline app/views/protocols/_participant_list.html.haml]: You should use data attributes to store data so: %button.btn.btn-primary.new-participant{type: "button", data: { protocol_id: protocol.id }, aria: {label: "Create Participant"}}

**Comment by kyle-glick** [comment]
I'll update the data attribute and add the format to the URL

**Comment by kyle-glick** [comment]
This is ready for review

**Comment by goleksiak** [comment]
Looks ready to merge

**Comment by ctt3** [comment]
Looks good to me

**Pull request merged**